src/helm/benchmark/export_scenario_text.py

```diff
@@ -0,0 +1,113 @@
+import json
+import os
+import argparse
+from typing import List
+from collections import defaultdict
+
+from helm.common.general import asdict_without_nones, ensure_directory_exists
+from helm.common.hierarchical_logger import hlog, htrack_block
+
+from helm.benchmark.scenarios.scenario import Scenario, create_scenario, Instance
+from helm.benchmark.presentation.run_entry import read_run_entries
+from helm.benchmark.run import run_entries_to_run_specs
+from helm.benchmark.runner import RunSpec
+from helm.benchmark.compute_contamination_metrics import LightInstance, LightScenario
+
+
+def create_light_instance_from_helm_instance(instance: Instance) -> LightInstance:
+    """Create a LightInstance given a helm Instance. Only keep the text attributes."""
+    input_text: str = instance.input.text
+    reference_texts: List[str] = [reference.output.text for reference in instance.references]
+    return LightInstance(input=input_text, references=reference_texts)
+
+
+def get_light_scenarios_from_helm(
+    run_spec: RunSpec, scenario_download_path: str = "exported_scenarios"
+) -> List[LightScenario]:
+    """Create a list of LightInstances given a helm run_spec. Only keep the text attributes."""
+
+    scenario: Scenario = create_scenario(run_spec.scenario_spec)
+
+    ensure_directory_exists(scenario_download_path)
+    scenario.output_path = os.path.join(scenario_download_path, scenario.name)
+    ensure_directory_exists(scenario.output_path)
+
+    scenario_name = scenario.name
+    if len(run_spec.scenario_spec.args) > 0:
+        scenario_name += f",{','.join([key+'='+value for key, value in run_spec.scenario_spec.args.items()])}"
```

Comment:
Keep the structured scenario spec (and definitely don't use 'name' to refer to something that's not just the name)

Reply:
What do you mean by "keep it structured". Are you suggesting using a dictionary? I am trying to avoid this so that `LightScenario` is actually light.

Reply:
As in, we would prefer `LightScenario` to contain `ScenarioSpec` rather than just a flat string (otherwise we'd have to parse the string to get back the original `ScenarioSpec`)

Reply:
OK, scenario_spec is now structured.